deformable_manipulation_experiment_params/launch/cloth_single_pole_params.launch

```diff
@@ -127,5 +140,11 @@
         <group ns="jacobian_controller">
             <param name="collision_scaling_factor"      value="1000.0"          type="double"/>
         </group>
+
+        <group ns="sampling_controller">
+            <param name="stretching_cosine_threshold"   value="$(arg stretching_cosine_threshold)"    type="double"/>
```

Comment:
Is there any reason there is an arg and a param version of these? The only utility for having both is to allow for math to be done with the value.